src/index.js

```diff
@@ -8,32 +8,12 @@ function main(client) {
 	let unappliedFunctions = new Set();
 	let appliedFunctions = new Set();
 
-	function retrieveEvolvCssAsset() {
-		const styleSheets = document.getElementsByTagName("link");
-
-		for (let i = 0; i < styleSheets.length; i++) {
-			const link = styleSheets[i];
-			if (link.href &&
-					link.href.indexOf('evolv.ai') >= 0 &&
-					link.href.indexOf('assets.css') >= 0) {
-				return link;
-			}
-		}
-
-		return undefined;
+	function retrieveEvolvCssAsset(environment) {
+		return document.querySelector(`link[href *= "${environment}"][href *= "assets.css"]`);
```

Comment:
You won't be able to use string interpolation (i.e. backticks) without a transpiler. Not supported in IE11 and older browsers.